app/public/icons/sandstorm/sandstorm.css

```diff
@@ -72,7 +72,7 @@
 .icon-approved_dark:before { content: "\f102"; }
 .icon-approved_light:before { content: "\f103"; }
 .icon-apps:before { content: "\f104"; }
-.icon-arrow-install:before { content: "\f105"; }
+.icon-arrow-install:before { content: "\f105"; line-height: normal; }
```

Comment:
Not related to this line, but - notice that in GitHub, you'll see this text:

"app/public/icons/sandstorm/sandstorm.css 100644 → 100755"

This means that Windows has changed the file permissions here in a meaningless way. I should probably add some `.gitattributes` files so that the permissions don't unnecessarily change.

Not a showstopper.